apps/wallet-dashboard/components/Dialogs/Staking/views/EnterAmountView.tsx

```diff
@@ -39,37 +45,45 @@ export interface FormValues {
 interface EnterAmountViewProps {
     selectedValidator: string;
     onBack: () => void;
-    onStake: () => void;
     showActiveStatus?: boolean;
-    gasBudget?: string | number | null;
     handleClose: () => void;
-    isTransactionLoading?: boolean;
+    amountWithoutDecimals: bigint;
+    senderAddress: string;
+    onSuccess?: (digest: string) => void;
 }
 
 function EnterAmountView({
     selectedValidator: selectedValidatorAddress,
     onBack,
-    onStake,
-    gasBudget = 0,
     handleClose,
-    isTransactionLoading,
+    amountWithoutDecimals,
+    senderAddress,
+    onSuccess,
 }: EnterAmountViewProps): JSX.Element {
     const coinType = IOTA_TYPE_ARG;
     const { data: metadata } = useCoinMetadata(coinType);
     const decimals = metadata?.decimals ?? 0;
 
+    const { addNotification } = useNotifications();
     const account = useCurrentAccount();
     const accountAddress = account?.address;
```

Comment:
we can remove the `accountAddress` because we have it already in the props as `senderAddress`